### ISSUE TITLE
"Continue the Conversation" suggestions sourced from unpublished articles

### ISSUE THREAD

**Issue by Algon-33** [StampyAI/stampy-chat#154]
In a chat I had with Stampy on the site recently, under "Continue the Conversation", there was a suggestion for "Do we have an example/evidence of the orthogonality thesis?" This is an unpublished article, and Stampy provided it anyway. The body of the response was "We don't have an answer for this question yet. Would you like to write one?" This seems like a bug. The suggestions should only be from articles which are live on the site. 

Example image: 
<img width="674" alt="Untitled" src="https://github.com/StampyAI/stampy-chat/assets/99098923/b516d6f8-f684-452c-a658-623045edd2e9">

**Comment by jrhender** [comment]
I also think this is a duplicate of #147 as @LeMurphant suggested here: https://github.com/StampyAI/stampy-chat/issues/147#issuecomment-2996734826 . Therefore, I propose this issue be closed or at least discussion be contained to #147 in order to make it easier to follow resolution efforts.

**Comment by jrhender** [comment]
@Algon-33 do you remember how you got to answer that had a followup suggestion of "Do we have an example/evidence of the orthogonality thesis?"?

**Comment by Algon-33** [comment]
No, I'm afraid I don't, but I managed to recreate this behaviour. I asked the following sequence of questions:

![Image](https://github.com/user-attachments/assets/c14f7a33-d170-4d0f-9edb-4e5a58cc1fa4)

![Image](https://github.com/user-attachments/assets/036f0bf0-7547-4d95-b040-f20c5cef72ab)

![Image](https://github.com/user-attachments/assets/187cd5d4-985b-4434-86e3-d4b1169c8235)

And got "Do we have an example/evidence of the orthogonality thesis?" as a suggestion.

![Image](https://github.com/user-attachments/assets/8ead1592-4c6a-4a3c-96b0-781793ea761c)


This was Stampy's response. 

![Image](https://github.com/user-attachments/assets/1d70b158-e06e-4ad8-92da-d4422eca227f)

**Comment by Algon-33** [comment]
> No, I'm afraid I don't, but I managed to recreate this behaviour. I asked the following sequence of questions:
> 
> ![Image](https://github.com/user-attachments/assets/c14f7a33-d170-4d0f-9edb-4e5a58cc1fa4)
> 
> ![Image](https://github.com/user-attachments/assets/036f0bf0-7547-4d95-b040-f20c5cef72ab)
> 
> ![Image](https://github.com/user-attachments/assets/187cd5d4-985b-4434-86e3-d4b1169c8235)
> 
> And got "Do we have an example/evidence of the orthogonality thesis?" as a suggestion.
> 
> ![Image](https://github.com/user-attachments/assets/8ead1592-4c6a-4a3c-96b0-781793ea761c)
> 
> This was Stampy's response.
> 
> ![Image](https://github.com/user-attachments/assets/1d70b158-e06e-4ad8-92da-d4422eca227f)

**Comment by Algon-33** [comment]
Closed as this is a duplicate of #147.
